<launch>
    <node pkg="active_panoptic_mapping" type="pano_seg_node.py" name="pano_seg_node" output="screen">
        <rosparam file="$(find active_panoptic_mapping)/config/pano_seg/mask2former.yaml"/>
        <remap from="~input_image" to="color_image" />  
        <remap from="~pano_seg" to="id_image" />  
    </node>

    <node pkg="habitat_ros" name="habitat_sim_node" type="habitat_sim_node.py" output="screen">
        <rosparam file="$(find habitat_ros)/config/rgbd.yaml" command="load" />
        <!-- <param name="scene_file" value="/media/giuliano/wd_ssd/datasets/habitat_test_data/habitat_test_scenes/versioned_data/habitat_test_scenes_1.0/apartment_1.glb" /> -->
        <param name="scene_file" value="/media/giuliano/wd_ssd/datasets/replica/apartment_0/habitat/mesh_semantic.ply" />
        <param name="enable_physics" value="false"/>
        <remap from="~rgb" to="color_image"/>
    </node>

    <node pkg="rviz" name="rviz" type="rviz" output="screen" args="-d $(find habitat_ros)/rviz/devel.rviz"/>
</launch>